### ISSUE TITLE
Several missing options in tandem adapter

### ISSUE THREAD

**Issue by oliverschilling** [OpenMS/OpenMS#1412]

**Comment by oliverschilling** [comment]
and once more with hopefully better formatting...

Dear all

I took a close look at the actual tandem parameter file that is generated by the X! tandem adapter. I noticed that a few options are missing or deactivated, without the option of activating them through the adapter (unless I overlooked it)
These include:

note type="input" label="scoring, cyclic permutation"
this is one of the better options in tandem; unfortunately it cannot be activated 
(it actually shows up twice in the parameter.xml file, wich is a bit strange as well)

note type="input" label="spectrum, minimum peaks"
cannot be set in the adapter, as far as I see

In the tandem parameter.xml, I cannot find the following parameters:
note type="input" label="protein, quick acetyl"
note type="input" label="protein, quick pyrolidone"

also missing is:
note type="input" label="spectrum, use conditioning"
which is also one of the better options of x! tandem

Thanks in advance for your support

Oliver
